src/components/Icons/Icon.stories.mdx

```diff
@@ -0,0 +1,276 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import Icon from "./Icon";
+import {
+  IconNames,
+  LogoNames,
+  IconColors,
+  IconSizes,
+  IconRotationTypes,
+} from "./IconTypes";
+import iconSvgs from "./IconSvgs";
+import List from "../List/List";
+import { ListTypes } from "../List/ListTypes";
+
+<Meta
+  title="Icon"
+  component={Icon}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=10734%3A3945",
+    },
+  }}
+/>
+
+# Icon
+
+<Description of={Icon} />
+
+<Preview withToolbar style={{ backgroundColor: "#fafafa" }}>
+  <Story
+    name="Icon Props Control"
+    args={{
+      name: "check",
+      color: IconColors.ui_black,
+      size: IconSizes.xlarge,
+      iconRotation: IconRotationTypes.rotate0,
+      decorative: true,
+      titleText: "An example icon",
+    }}
+    argTypes={{
+      color: {
+        control: {
+          type: "select",
+          options: IconColors,
+        },
+      },
+      size: {
+        control: {
+          type: "select",
+          options: IconSizes,
+        },
+      },
+      iconRotation: {
+        control: {
+          type: "select",
+          options: IconRotationTypes,
+        },
+      },
+      blockName: {
+        table: {
+          disable: true,
+        },
+      },
+      modifiers: {
+        table: {
+          disable: true,
+        },
+      },
+    }}
+  >
+    {args => <Icon {...args} />}
+  </Story>
+</Preview>
+
+<ArgsTable story="Icon Props Control" />
+
+export const iconRow = (icon, opts = {}) => {
```

Comment:
ooooh! I totally thought the icons were literally black boxes haha 
Can we use a not-black colour (ui-gray-xdark) attacht he background color to the `<li>` element rather than the icon? I think this might give a better sense of what's going on.  

![Screen Shot 2021-07-06 at 9 04 09 AM](https://user-images.githubusercontent.com/3579472/124604731-39d9a880-de39-11eb-9c89-9cb6239e3d5a.png)

Not really functionality related though, so totally put it to another ticket

Reply:
Yeah, I pointed that above that all the library-svg logos are really big in size. We'll have to create a ticket to discuss including them and then fixing the sizes because right now the largest css modifier is not big enough to display the whole svg.